src/test/java/nl/tudelft/contextproject/logging/TestLog.java

```diff
@@ -78,8 +72,7 @@ public void verifyLogged(Level level, String msg, Exception ex) {
 	@Test
 	public void testLog() throws IOException {
 		Log nlog = new Log("hello", true);
-		
-		//There should be a filehandler and a consolehandler.
```

Comment:
For once my comment doesn't give the same information as the code. This one should be readded, as it explains why we expect 2 instead of 42.